### ISSUE TITLE
API status page for users

### ISSUE THREAD

**Issue by StephenChan** [beijbom/coralnet#294]
Including the following info:

- Rate limiting stats for the user. (Brought up in issue #277)
- Status of API jobs requested by the user. Could include all in-progress jobs + completed jobs from the past 30 days. (Related to issue #293, the dev-facing job monitor)

Where should this status page be linked from?

**Comment by beijbom** [comment]
Good question. It seems like we could include a summary dashboard on the source main page. This would list deploy jobs associated with this source (submitted by any user).

We could then add user-centric view on the user profile page. This would include jobs submitted by that user (related to any source)

**Comment by StephenChan** [comment]
> Good question. It seems like we could include a summary dashboard on the source main page. This would list deploy jobs associated with this source (submitted by any user).

Hmm... interesting idea, but I think it would have to be filtered somehow. As far as getting status information, I don't think people generally need to see the status of other users' jobs. Unless it's one source member seeing another source member's jobs (since they're likely working together). There's also a chance that public-source owners wouldn't want to have a prominent feed on their source which is potentially filled with strangers' activity. Unless it's just a popularity statistic, like "This source's classifiers have been deployed on 7,456 images."

> We could then add user-centric view on the user profile page. This would include jobs submitted by that user (related to any source)

Yeah, I think we should definitely have this, but I just hope it's an obvious enough place to look. Currently you'd have to click a button that says "Account (stephen)" for example.

I know many websites (GitHub, Amazon, etc.) have a dropdown next to your name to show various options: Settings, Profile, Sign out, etc. Maybe it's time for us to do something similar, and add 'API usage' or 'API jobs' as one of the dropdown items?

**Comment by beijbom** [comment]
> Unless it's just a popularity statistic, like "This source's classifiers have been deployed on 7,456 images."

Yeah, I agree. This is actually what I was thinking. Just a nugget of information to make them feel good about how much the source is used :)

> I know many websites (GitHub, Amazon, etc.) have a dropdown next to your name to show various options: Settings, Profile, Sign out, etc. Maybe it's time for us to do something similar, and add 'API usage' or 'API jobs' as one of the dropdown items?

Yeah, sounds good. Depending on other priorities we could start simple during beta-testing and then make it fancier later.
